.github/workflows/static-tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           "exam-manager",
           "workflow-manager",
           "device-manager",
-          "acquisition-control",
+          "acquisition_control",
```

Comment:
Isn't this just the designation of the microservice? Does the pep guideline apply here?
If we decide to use the underscore convention we should also apply it to the other microservices to be consistent.

Reply:
Otherwise you can't import from that path in python. I would recommend to change it everywhere. Maybe we should discuss about it again.

Reply:
I see, but then we should set up a designated shared library, that different services can import from.